flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -105,18 +105,27 @@
 
 	/**
 	 * This option specifies the failover strategy, i.e. how the job computation recovers from task failures.
+	 * The option "individual" and "region-legacy" are intentionally not included
+	 * as they have some known limitations or issues.
+	 * "individual" strategy only works when all tasks are not connected, in which case the "region"
+	 * failover strategy would also restart failed tasks individually.
+	 * "region-legacy" strategy is not able to backtrack missing input result partitions.
+	 * The new "region" strategy supersedes "individual" and "region-legacy" strategies and should always work.
 	 */
-	@Documentation.ExcludeFromDocumentation("The failover strategy feature is highly experimental.")
 	public static final ConfigOption<String> EXECUTION_FAILOVER_STRATEGY =
 		key("jobmanager.execution.failover-strategy")
 			.defaultValue("full")
 			.withDescription(Description.builder()
 				.text("This option specifies how the job computation recovers from task failures. " +
 					"Accepted values are:")
 				.list(
-					text("'full': Restarts all tasks."),
-					text("'individual': Restarts only the failed task. Should only be used if all tasks are independent components."),
-					text("'region': Restarts all tasks that could be affected by the task failure.")
+					text("'full': Restarts all tasks to recover the job."),
+					text("'region': Restarts all tasks that could be affected by the task failure. " +
```

Comment:
Maybe we can omit some details here, and place a hyperlink to the *Task Failure Recovery* page using `LinkElement`. WDYT?

Reply:
Ok. Let's make it this way.